### ISSUE TITLE
Please add option to hide menu bar icon

### ISSUE THREAD

**Issue by CollinsVakayil** [mattDavo/Yippy#25]
For those of us that use the shortcut to trigger Yippy clipboard, we'd like an option to hide the menu bar icon.
Also please add a button to clear clipboard history when the clipboard window is open.

**Comment by albert239825** [comment]
Yes please! The functionality of Yippy is amazing, but it serves little purposing taking up menu bar icon space. Thank you so much to the developers!
